src/unix/mediactrl_gstplayer.cpp

```diff
@@ -337,6 +375,18 @@ bool wxGStreamerMediaBackend::CreateControl(wxControl* ctrl, wxWindow* parent,
     m_video_renderer = gst_player_video_overlay_video_renderer_new(window_handle);
     m_player = gst_player_new(m_video_renderer, gst_player_g_main_context_signal_dispatcher_new(NULL));
 
+    wxDisplayInfo info = wxGetDisplayInfo();
+    if (info.type == wxDisplayWayland) {
```

Comment:
Another style nitpick
```suggestion
    if (info.type == wxDisplayWayland)
    {
```